golang/sqlboiler/templates/00_struct.tpl

```diff
@@ -143,3 +144,51 @@ func (o {{$modelName}}) {{titleCase $column.Name}}() string {
 }
 {{- end -}}
 {{- end }}
+
+{{/* Add resolvers to FK relationships */}}
+{{range .Table.FKeys -}}
+{{- $txt := txtsFromFKey $dot.Tables $dot.Table . -}}
+// {{$txt.ForeignTable.NameGo}} pointed to by the foreign key
+func (o {{$modelName}}) {{$txt.ForeignTable.NameGo}}() *{{$txt.ForeignTable.NameGo}} {
+  if o.model.R == nil || o.model.R.{{$txt.Function.Name}} == nil {
+    return nil
+  }
+  return &{{$txt.ForeignTable.NameGo}}{
+		model: *o.model.R.{{$txt.Function.Name}},
+		db:    o.db,
+	}
+}
+{{end -}}
```

Comment:
Resolved the relationship by getting the data from the _relationships_ field. Will not be `nil` if eager loading was specified when retrieving `o.model`